.drone.yml

```
@@ -172,6 +199,29 @@ data:
           - frontend.deployment.tag=$DRONE_COMMIT_SHA
           - botTwitter.deployment.tag=$DRONE_COMMIT_SHA
 
+    - name: deploy-production.auction-ui.k8s.sidestream.tech
```

Is it actually required to have no spaces in the step name? (: 
```suggestion
    - name: Deploy production.auction-ui.k8s.sidestream.tech
```

---

It's not, I would prefer it though as these names are also used in other parts of the code (like declaring dependencies on these steps). So to keep it all consistent and avoid unexpected surprises, i'd like to stay with `all-lower-case-dash-naming`